src/core/reactor.cc

```diff
@@ -850,6 +851,9 @@ static void print_with_backtrace(backtrace_buffer& buf, bool oneline) noexcept {
         buf.append_decimal(this_shard_id());
     }
 
+    buf.append(", in scheduling group ");
```

Comment:
It should happen inside `if (local_engine)` check above

Reply:
Updated.